COMPILE_MSVC.TXT

```diff
@@ -106,3 +106,17 @@ versions, and Windows Driver Kit 8.1 Update 1 or newer versions are required.
 
       >sc delete test_winkernel
       >bcdedit /deletevalue testsigning
+      
+      
+      
+(3) Installing and building capstone via vcpkg
+  
+  You can download and install capstone using the vcpkg(https://github.com/Microsoft/vcpkg) dependency manager:
+
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    vcpkg install capstone
```

Comment:
This disables support for all architectures supported by capstone. You need to explicitly enable them like this:
```
vcpkg install capstone[core,x86,ppc]:x64-windows
```

Reply:
oh, the "core" should not need to be specified, but ON by default, no?

Reply:
@ceeac  and @aquynh 
Thanks for your review.
Since this port is special. We can add any one of features listed in [CONTROL ](https://github.com/microsoft/vcpkg/blob/master/ports/capstone/CONTROL)file to build `capstone`, which depends on your requirement. As for `core`, there is no need to add additionally.